### PR TITLE
Add support for custom HTTP host header.

### LINKED ISSUE #76: Can't send a Host header
I think this is a Go HTTP limitation (I've seen it before), but you can't pass a Host header:

```
[#] boom -n 1 -c 1 -h Host:google.com http://requestb.in/1mkzwmn1

HEADERS

Via: 1.1 vegur
Host: requestb.in
Content-Type: text/html
Connection: close
Accept-Encoding: gzip
Connect-Time: 3
Total-Route-Time: 0
User-Agent: Go 1.1 package http
X-Request-Id: f2e7cd0f-995e-41f5-9258-b025a579314f
```

```
[#] boom -n 1 -c 1 -h Blah:google.com http://requestb.in/1mkzwmn1

Blah: google.com
Host: requestb.in
Via: 1.1 vegur
Content-Type: text/html
Connection: close
Accept-Encoding: gzip
Connect-Time: 14
Total-Route-Time: 0
User-Agent: Go 1.1 package http
X-Request-Id: cad35d5f-9c2a-4b79-99c1-afae0dd7eade
```

### PR DESCRIPTION
**Pull request by espebra**
Thank you for a nice tool. I need to set a custom host header, so I introduced the -H parameter for this in my fork. Hopefully it will be useful for others.

Please note that the import path github.com/espebra/boom/boomer in boom.go must be changed back to github.com/rakyll/boom/boomer if pulled. Fixes #76.

### CODE REVIEW

**Comment by matschaffer** [comment]
This branch worked for me. Would be nice to get rebased/merged. Looks like some of the CLI switches have changed since last September so I had to tweak my command a bit for this branch:

```
./boom -H '...' -h 'Cookie:key=val' -n 50 -c 1 'http://127.0.0.1:3000/...'
```

**Comment by missedone** [comment]
hi @matschaffer 
this feature was merged to master, see PR #89 , note that option '-h' was deprecated with in favor of option '-H'

**Comment by matschaffer** [comment]
@missedone maybe there's some confusion here.

The feature I was looking for was the `Host:` override, which master doesn't let you set using the `-H` option.

**Comment by missedone** [comment]
oh, i read it too fast, 
yes, you want to override the "Host: ".

i think @espebra need to rebase the PR.

**Comment by espebra** [comment]
I'll be glad to rebase if it is likely that the pull request eventually will be accepted. However, seeing that there are many outstanding pull requests I don't think it's very likely unfortunately.

File: boom.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/rakyll/boom/boomer"
+	"github.com/espebra/boom/boomer"
```
Comment (rakyll) [inline boom.go]: Could you switch it back to rakyll?

**Comment by espebra** [comment]
Sure, I've done that now. Would you like me to rebase?

**Comment by rakyll** [comment]
The project has moved to https://github.com/rakyll/hey. Closing the PRs on this legacy repo.

**Pull request closed without merging**